src/index.ts

```diff
@@ -0,0 +1,125 @@
+
+const MONTHS_IN_YEAR = 12;
+const MINIMUM_PURCHASE_AMOUNT = 800;
+
+export type Result = {
```

Comment:
🔧 The name `Result` is too generic, it's better to name it more concrete because it can be imported and used outside of that library

Reply:
it's changed to `DepreciationResult` https://github.com/kontist/asset-depreciation-calculator/pull/1/commits/cf6ba5a84787cbd5870f4c10f2e2be6b4c18ab8c